setup.cfg

```diff
@@ -12,6 +12,7 @@ extend-ignore = E203,E501
 disallow_untyped_defs = true
 exclude = tests/system/migrations
 namespace_packages = true
+no_implicit_optional = false
```

Comment:
didn't found that option

Reply:
Funny that it still works, seems like it is just deprecated. But I will change our code base to use strict optional then.